Ambiguous (glob) import on loongarch64

Builds on `loongarch64` fail due to ambiguous imports. This is due to multiple glob imports matching items with the same name:

```
error[E0659]: `STATX_TYPE` is ambiguous
   --> /home/buildozer/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/rustix-1.0.8/src/fs/statx.rs:73:25
    |
73  |         const TYPE = c::STATX_TYPE;
    |                         ^^^^^^^^^^ ambiguous name
    |
    = note: ambiguous because of multiple glob imports of a name in the same module
note: `STATX_TYPE` could refer to the constant imported here
   --> /home/buildozer/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/rustix-1.0.8/src/backend/libc/c.rs:7:16
    |
7   | pub(crate) use libc::*;
    |                ^^^^^^^
    = help: consider adding an explicit import of `STATX_TYPE` to disambiguate
note: `STATX_TYPE` could also refer to the constant imported here
   --> /home/buildozer/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/rustix-1.0.8/src/backend/libc/c.rs:512:16
    |
512 | pub(crate) use statx_flags::*;
    |                ^^^^^^^^^^^^^^
    = help: consider adding an explicit import of `STATX_TYPE` to disambiguate
```

Full log output: https://gitlab.alpinelinux.org/WhyNotHugo/aports/-/jobs/1993607#L798

--- a/src/backend/libc/c.rs
+++ b/src/backend/libc/c.rs
@@ -490,7 +490,11 @@ pub(super) use readwrite_pv64v2::{preadv64v2 as preadv2, pwritev64v2 as pwritev2
 #[cfg(all(
     linux_like,
     linux_raw_dep,
-    not(any(target_os = "emscripten", target_env = "gnu"))
+    not(any(
+        target_os = "emscripten",
+        target_env = "gnu",
+        all(target_arch = "loongarch64", target_env = "musl")
+    ))
 ))]
 mod statx_flags {
     pub(crate) use linux_raw_sys::general::{
@@ -509,7 +513,11 @@ mod statx_flags {
 #[cfg(all(
     linux_like,
     linux_raw_dep,
-    not(any(target_os = "emscripten", target_env = "gnu"))
+    not(any(
+        target_os = "emscripten",
+        target_env = "gnu",
+        all(target_arch = "loongarch64", target_env = "musl")
+    ))
 ))]
 pub(crate) use statx_flags::*;
 

